frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/GitMacheteBundle.java

```diff
@@ -0,0 +1,27 @@
+package com.virtuslab.gitmachete.frontend.actions.common;
+
+import java.util.function.Supplier;
+
+import com.intellij.DynamicBundle;
+import org.jetbrains.annotations.PropertyKey;
+
+public final class GitMacheteBundle extends DynamicBundle {
```

Comment:
Hmm I don't like the `actions.messages.GitMacheteBundle` vs `actions.ResourceBundle` distinction... it's actually not clear from the names why they should be split that way (if they should be split at all). I didn't change this on #422, TBD on this PR